plugins/admin.py

```diff
@@ -0,0 +1,131 @@
+"""admin actions
+
+Commands:
+!kick
+!ban
+!unban
+
+# ../config.py
+# int | str | list
+ADMIN_CHATS = -1234567890123
+"""
+import asyncio
+from time import time
+from pyrogram import Client, filters
+from pyrogram.types import Message
+from pyrogram.errors import UserNotParticipant
+from config import ADMIN_CHATS
+
+USAGE_KICK = """**Usage**:
+*admin reply to a user message*
+`!ban [reason]`
+"""
+USAGE_BAN = """**Usage**:
+*admin reply to a user message*
+`!ban [reason]`
+"""
+USAGE_UNBAN = """**Usage**:
+*admin reply to a user message*
+`!unban [reason]`
+"""
+
+# should be more than 30 seconds
+# or less than 366 days
+DURATION_UNTIL_UNBAN = 60
+DELAY_DELETE = 15
+
+
+@Client.on_message(filters.command(["kick"], prefixes="!")
+                   & filters.chat(ADMIN_CHATS)
+                   & filters.incoming
+                   & ~filters.edited)
+async def kick_user(_, message: Message):
+    """!kick admin kick user"""
+    try:
+        if not (await _should_allow_admin_action(message)):
```

Comment:
hm i think would be a bit more readable if it was like this:

`if(await _admin_action_not_allowed(message)):`

with no `not`

because if you always reverse the boolean value that the method returns why not return already reversed value